packages/python-vtapi3/PKGBUILD

```diff
@@ -0,0 +1,38 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-vtapi3
+_pkgname=virustotalapi3
+pkgver=259.7d9e278
+pkgrel=1
+pkgdesc='The module that implements the VirusTotal API functions (3 version).'
+arch=('any')
+url='https://github.com/drobotun/virustotalapi3'
+license=('MIT')
+depends=('python' 'python-requests')
+makedepends=('python-setuptools')
+options=(!emptydirs)
+source=("git+https://github.com/drobotun/$_pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $_pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+build() {
+  cd "$_pkgname"
+
+  python setup.py build
+}
+
+package() {
+  cd "$_pkgname"
+
+  python setup.py install --root="$pkgdir" --prefix=/usr -O1 --skip-build
+  
+  rm -rf "$pkgdir/usr/lib/python3.10/site-packages/tests/__init__.py"
```

Comment:
> Juste keep `rm -rf "$pkgdir/usr/lib/python3.10/site-packages/tests/` and put it before `python setup.py install`

These files should be generated by `python setup.py install` because if I put `rm -rf` command before it, I still get the error.

Reply:
Ok so just group the 3 rm.

Reply:
cf. https://wiki.archlinux.org/title/Python_package_guidelines#Test_directory_in_site-package and https://github.com/drobotun/virustotalapi3/issues/3